swig/sattools_swig0.i

```diff
@@ -5,19 +5,16 @@
 %include "gnuradio.i"			// the common stuff
 
 //load generated python docstrings
-%include "sattools_swig_doc.i"
+%include "sattools_swig0_doc.i"
 
 %{
-#include "sattools/csp_zmqhub_pub_sink.h"
 #include "sattools/reed_solomon_ccsds_decode.h"
 #include "sattools/async_crc32c.h"
 #include "sattools/golay24_decode.h"
 #include "sattools/asm_golay_decoder.h"
 %}
 
 
-%include "sattools/csp_zmqhub_pub_sink.h"
```

Comment:
Does it not work to flag out the zeromq stuff using `#IFDEF` with a define passed into swig? Then we wouldn't need the complexity of all these files I think.